### ISSUE TITLE
Bokeh not plotting the official website snippet

### ISSUE THREAD

**Issue by jsulopzs** [kernc/backtesting.py#926]
### Actual Behavior

You may watch the following Loom video to understand the possible error:

https://www.loom.com/share/a078b84b92ba46feb1a9abf2d8fc8b93

### Steps to Reproduce

1. Install backtesting library `pip install backtesting`
2. Copy-paste a snippet of code from the [official website](https://kernc.github.io/backtesting.py/).
3. Execute the code.

```python

from backtesting import Backtest, Strategy
from backtesting.lib import crossover

from backtesting.test import SMA, GOOG


class SmaCross(Strategy):
    n1 = 10
    n2 = 20

    def init(self):
        close = self.data.Close
        self.sma1 = self.I(SMA, close, self.n1)
        self.sma2 = self.I(SMA, close, self.n2)

    def next(self):
        if crossover(self.sma1, self.sma2):
            self.buy()
        elif crossover(self.sma2, self.sma1):
            self.sell()


bt = Backtest(GOOG, SmaCross,
              cash=10000, commission=.002,
              exclusive_orders=True)

output = bt.run()
bt.plot()

```

### Additional info

- Backtesting version: 0.3.3   
- `bokeh.__version__`: 3.0.3
- OS: GitHub codespaces

**Comment by jsulopzs** [comment]
Already solved in #803

**Comment by mr-september** [comment]
Bad practice to mark issues as completed and claim "solved" when they aren't (as of writing this comment, #803 is still open and unsolved), you'll just get more duplicates and frustrated users.
